bootstrap_salt/fab_tasks.py

```diff
@@ -107,6 +126,30 @@ def get_master_ip():
 
 
 @task
+def swap_masters(tag1, tag2):
+    """
+    Swap two tagged stacks.
+    """
+    cfn_config = get_config()
+    r53_conn = get_connection(R53)
+    zone_name = cfn_config.data['master_zone']
```

Comment:
Since this is a user accessible task, it might be good to catch 'no master zone' and output some more helpful text here